src/NServiceBus.AcceptanceTests/Audit/When_a_replymessage_is_audited.cs

```diff
@@ -75,7 +77,7 @@ public class MessageToBeAuditedHandler : IHandleMessages<ResponseToBeAudited>
 
                 public Task Handle(ResponseToBeAudited message, IMessageHandlerContext context)
                 {
-                    Assert.AreEqual(context.MessageHeaders["MyHeader"], "SomeValue");
+                    TestContext.HeaderValue = context.MessageHeaders["MyHeader"];
```

Comment:
👍 for taking all those asserts out and temporarily storing on `TestContext`.
To be more clear, should `TestContext.HeaderValue` be named specifically for the context? I.e. `TestContext.HeaderValueReceivedByMessageToBeAuditedHandler`? 

The general `HeaderValue` is good when the context is fresh in our heads. A few weeks for now I'll have to think longer than a few seconds to recall what is what 🙂